azkaban-web-server/src/main/java/azkaban/scheduler/QuartzScheduler.java

```diff
@@ -148,23 +126,32 @@ public synchronized boolean isJobPaused(final String jobName, final String group
   }
 
   /**
-   * resume a paused job given the groupname. since resuming request might be issued concurrently,
-   * so synchronized is added to ensure thread safety.
+   * Resume a job.
+   * @param jobName
+   * @param groupName
+   * @return true the job has been resumed, no if the job doesn't exist.
+   * @throws SchedulerException
    */
-  public synchronized void resumeJob(final String jobName, final String groupName)
+  public synchronized boolean resumeJobIfPresent(final String jobName, final String groupName)
       throws SchedulerException {
-    checkJobExistence(jobName, groupName);
-    this.scheduler.resumeJob(new JobKey(jobName, groupName));
+    if (ifJobExist(jobName, groupName)) {
+      this.scheduler.resumeJob(new JobKey(jobName, groupName));
+      return true;
+    } else {
+      return false;
+    }
   }
 
   /**
-   * Unregister a job given the groupname. Since unregister might be called when
-   * concurrently removing projects, so synchronized is added to ensure thread safety.
+   * Unschedule a job.
+   * @param jobName
+   * @param groupName
+   * @return true if job is found and unscheduled.
+   * @throws SchedulerException
    */
-  public synchronized void unregisterJob(final String jobName, final String groupName) throws
+  public synchronized boolean unschedule(final String jobName, final String groupName) throws
```

Comment:
How about unscheduleJob. The method name better to be consistent.

Reply:
udpated